api/History.json

```diff
@@ -314,6 +359,51 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "title": {
+          "__compat": {
+            "description": "Whether the <code>title</code> argument is used",
+            "support": {
+              "chrome": {
+                "version_added": false
```

Comment:
Chrome, Chrome Android, and WebView Android support this argument. 

https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/frame/history.idl

Reply:
Did you read https://github.com/whatwg/html/issues/2174? It explains that it's a no-op and there isn't "supported". I'm documenting this to avoid the issues described in https://github.com/whatwg/html/issues/2174#issuecomment-403521462

Reply:
Diving through the code reveals that the `title` argument, even though it's passed in, [is not actually used](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/frame/history.cc?g=0&l=266) in the underlying function call.